<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-S95UV7N" ignored="0" name="Test" tests="8" failures="1" timestamp="02 Jan 2019 06:56:48 GMT" time="25.023" errors="0">
  <testcase name="launchFlightReservationApplication" time="4.159" classname="testcases.ExcuteTest"/>
  <testcase name="testFail" time="0.0" classname="testcases.ExcuteTest">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at testcases.ExcuteTest.testFail(Unknown Source)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- testFail -->
  <testcase name="testSkip" time="0.0" classname="testcases.ExcuteTest">
    <skipped/>
  </testcase> <!-- testSkip -->
  <testcase name="verifyLinks" time="1.521" classname="testcases.ExcuteTest"/>
  <testcase name="verifyWebTable" time="2.044" classname="testcases.ExcuteTest"/>
  <testcase name="login" time="7.435" classname="testcases.ExcuteTest"/>
  <testcase name="reserveAFlight" time="3.657" classname="testcases.ExcuteTest"/>
  <testcase name="bookATicket" time="2.045" classname="testcases.ExcuteTest"/>
</testsuite> <!-- Test -->
